lab-michael/test/test-greeting.js

```diff
@@ -1,9 +1,11 @@
-var greet = require('../greeting.js');
+var greet = require('../lib/greeting.js');
 var assert = require('assert');
 
-describe('A greeting', function() {
-  it('will greet user by name', function() {
-    var name = 'bob';
-    assert.equal(greet.greet(name), 'hello ' + name);
+describe('func', function() {
```

Comment:
This describe block should be a little more descriptive; i.e. `the greet module`. The next nested block should name the method, and then the it blocks are individual tests.